### ISSUE TITLE
Option for fonts?

### ISSUE THREAD

**Issue by ChristoferMinestar** [ImpactDevelopment/ImpactClient#680]
Option for loading custom .ttf fonts like Flux? Thanks

**Comment by ChristoferMinestar** [comment]
[Enhancement] [Long-term]

**Comment by LordQuadrato** [comment]
Do you mean like a custom font for the ClickGui and HUD text in the client? Or do [custom chat fonts](https://github.com/ImpactDevelopment/ImpactClient/issues/570#issuecomment-408561504) already describe your issue?

**Comment by LeafHacker** [comment]
cChat doesn't support custom TTF fonts serverside, but anything should be possible clientside.

Are you asking for the Impact font used in HUD, ClickGUI, etc to be customisable or the Minecraft font used in vanilla GUIs, signs, etc? Or both?

How do you envision a user would change the font?

**Comment by ChristoferMinestar** [comment]
Yeah, like custom ttf fonts for hud, click GUI, and possibly client GUI.
